README.md

```diff
@@ -823,14 +823,25 @@ The opposite `$nin` is also available.
 
 The following numeric comparisons operators are supported: `$lt`, `$lte`, `$gt`, `$gte`.
 
-The `$exists` operator matches document containing the field, even if this field is `null`
+The `$exists` operator matches documents containing the field, even if this field is `null`
 
 ```json
 {"type": {"$exists": true}}
 ```
 
 You can invert the operator by passing `false`.
 
+There is also a `$regex` operator that matches documents containing the field given as a regular expression. The syntax of the regular expressions accepted is the same general syntax used by Perl, Python, and other languages.
+More precisely, it is the syntax accepted by RE2 and described at https://golang.org/s/re2syntax, except for \C.
```

Comment:
I think we should say that the accepted syntax may vary depending on the storage handler.